src/updatedownloader.cpp

```diff
@@ -173,7 +102,7 @@ void UpdateDownloader::Run()
 
     try
     {
-      const std::wstring tmpdir = CreateUniqueTempDirectory();
+      const std::wstring tmpdir = winsparkle::CreateUniqueTempDirectory();
```

Comment:
Is name space designation `winsparkle::` really needed here when already inside the namespace?